fbw-a380x/src/systems/instruments/src/EWD/elements/ThrustGauge.tsx

```diff
@@ -394,8 +420,8 @@ export class ThrustGauge extends DisplayComponent<ThrustGaugeProps> {
             <GaugeThrustComponent
               x={this.props.x}
               y={this.props.y}
-              valueIdle={0.04}
-              valueMax={2.6}
+              valueIdle={Subject.create(0.04)}
```

Comment:
Not sure if this is good practice or not in the avionics framework, maybe someone else can comment on that

Reply:
improved in latest version, after input from tracer :)